qa/tasks/deepsea.py

```diff
@@ -45,6 +46,16 @@ def dump_file_that_might_not_exist(remote, fpath):
         pass
 
 
+def check_ceph_test_rpm_version(remote):
```

Comment:
please make this function a method in `class Orch`. The name should be `__check_ceph_test_rpm_version`.

Please note that the methods are in alphabetical order in the code, so this one should be inserted between `__ceph_health_test` and `__check_salt_api_service`.

Reply:
Done